src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -593,11 +597,16 @@ public void selectDropdownByActualValue(WebElement element, String value) {
     }
 
     /**
-     * Returns the status message in the page. Returns "" if there is no
-     *         status message in the page.
+     * Returns a list of Texts of user status messages in the page.
```

Comment:
`Returns a list of the text of user status messages in the page.` is sufficient, no need to mention that an empty list will be returned.

Also, add `@see WebElement#getText()`.

Reply:
I think you missed this: `Returns a list of the text of user status messages in the page.`

`Texts` should not be capitalized.